euler/README.md

```diff
@@ -0,0 +1,263 @@
+28.10.2016  09:35    <DIR>          inputs
+28.10.2016  09:35    <DIR>          outputs
+               0 File(s)              0 bytes
+               4 Dir(s)  528'803'373'056 bytes free
+
+[esri104] darthoma@ITA-SCHLUE-W-17 C:\Users\darthoma\Documents\GitHub\CEAforArcGIS
+> dir %temp%\samples
+ Volume in drive C has no label.
+ Volume Serial Number is C84C-1BEB
+
+ Directory of C:\Users\darthoma\AppData\Local\Temp\samples
+
+04.11.2016  09:51    <DIR>          .
+04.11.2016  09:51    <DIR>          ..
+04.11.2016  09:34             1'943 problem.pickle
+04.11.2016  09:49               329 result.0.csv
+04.11.2016  09:49               330 result.1.csv
+04.11.2016  09:51               330 result.10.csv
+04.11.2016  09:51               330 result.11.csv
+04.11.2016  09:49               330 result.2.csv
+04.11.2016  09:49               330 result.3.csv
+04.11.2016  09:50               330 result.4.csv
+04.11.2016  09:50               330 result.5.csv
+04.11.2016  09:50               331 result.6.csv
+04.11.2016  09:50               331 result.7.csv
+04.11.2016  09:50               319 result.8.csv
+04.11.2016  09:50               330 result.9.csv
+04.11.2016  09:34             1'136 samples.npy
+              14 File(s)          7'029 bytes
+               2 Dir(s)  528'819'286'016 bytes free
+
+[esri104] darthoma@ITA-SCHLUE-W-17 C:\Users\darthoma\Documents\GitHub\CEAforArcGIS
+> type %TEMP%\samples\result.0.csv
+,QHf_MWhyr,QCf_MWhyr,Ef_MWhyr,QEf_MWhyr
+0,381.043,18.824,156.259,556.126
+1,381.21,19.017,156.256,556.482
+2,381.068,18.86,156.256,556.184
+3,381.175,19.12,156.256,556.551
+4,381.338,19.943,156.249,557.53
+5,380.992,18.899,156.249,556.14
+6,381.998,21.714,156.336,560.048
+7,381.367,20.169,156.255,557.792
+8,0.0,0.0,0.0,0.0
+```
+
+The above steps run morris sampling (the default) with N=1, grid_jump=2 (default) and num_levels=4 (default), simulates
+all the samples and stores the results back into the sampling directory.
+
+On the Euler cluster, we can set N to a much higher number, say, 1000 and we would then
+run `sensitivity_demand_simulate` multiple times, incrementing the `--sample-index`
+parameter by `--number-of-simulations` each time. We need to choose `--number-of-simulations`
+to fit the simulation count in the time slot of the node as the Euler cluster will kill
+any process that runs longer than a fixed amount of time. Keeping it as large as possible
+reduces the overhead of waiting for a node to pick up the job, so we will need to
+experiment a bit here... The bash scripts for the euler cluster do this automatically and have a batch size of 100,
+which works well for the `reference-cas-open`.
+
+The next step is to run the analysis on the results. This is done in a single process.
+
+```
+[esri104] darthoma@ITA-SCHLUE-W-17 C:\Users\darthoma\Documents\GitHub\CEAforArcGIS
+> python cea\analysis\sensitivity\sensitivity_demand_analyze.py --samples-folder %TEMP%\samples
+```
+
+### Running on Euler
```

Comment:
Is there a procedure to use the nine_cubes.zip case study we have in the repository? i think it would be good to use this location to run the simulations in euler?

btw, is there a way to send folders and their contents from windows to euler?